src/domain/task.go

```diff
@@ -0,0 +1,58 @@
+package domain
+
+import (
+	uuid "github.com/satori/go.uuid"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+type TaskInstance struct {
+	Id        uuid.UUID `json:id`
```

Comment:
These field names need quoting (I think that'll fix the capitalisation in the JSON responses)

Reply:
ah yes! good spot